core/src/main/java/com/datastax/oss/driver/api/core/data/CqlVector.java

```diff
@@ -56,36 +62,48 @@ public int hashCode() {
 
   @Override
   public String toString() {
+    StringBuilder builder = new StringBuilder("CqlVector{");
+    for (T value : values) {
+      builder.append(value).append(", ");
```

Comment:
nit: Consider `String.join()`?

Reply:
I tried this, but the generic parameter broke it, as there is no bound on T which makes it a CharSequence.